src/emulator/gui/include/gui/state.h

```diff
@@ -58,7 +58,7 @@ struct GuiState {
 
     DialogState common_dialog;
     GamesSelector game_selector;
-    MemoryEditor *memory_editor;
+    MemoryEditor *memory_editor = nullptr;
```

Comment:
Next time use braced initialization please, as we do everywhere else. Or better yet, use smart pointers.
```cpp
    MemoryEditor *memory_editor{};
```